### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by paulinebessoles**
#### :tophat: What? Why?
This PR is related to the documentation update. 

#### :tophat: Done

- [x] Create template page
- [x] Create subpages
- [x] Update navigation
- [x] Update screenshots

#### Testing

:hearts: Thank you!

### CODE REVIEW

File: docs/en/modules/admin/pages/templates/proposal_answers.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
+= Proposal answers templates
+
+In this section, you can create, edit, and delete proposal answers templates, that you can then use when answering proposals. 
+
+== Create a proposal answers template
+
+NOTE: To be able to create a proposal answers template, you need to have created at least one 
+xref:admin:components/proposals.adoc[proposals] component.
+
+To create a new block proposal answers template, you need to click the "New template" button.
```
Comment (carolromero) [inline docs/en/modules/admin/pages/templates/proposal_answers.adoc]:
```suggestion
To create a new proposal answers template, you need to click the "New template" button. 
```

File: docs/en/modules/admin/pages/templates/proposal_answers.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+image::templates/templates_proposal_answers_backend_form.png[Backend: new proposal answer template form]
+
+.New proposal answer template form
+|===
+|Field |Type |Description
+
+|Name
+|Required
+|Title of this answer template. Choose a clear title, describing precisely what the answer is for, like "accepted proposal."
```
Comment (carolromero) [inline docs/en/modules/admin/pages/templates/proposal_answers.adoc]:
```suggestion
|Title of this answer template. Choose a clear title, describing precisely what the answer is for, like "accepted proposal".
```

File: docs/en/modules/admin/pages/templates/proposal_answers.adoc [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+|Proposal state
+|Required
+|By default, you can choose Evaluating, Accepted, or Rejected. 
+
+|===
+
+== Manage
+
+All the templates appear in the questionnaire templates list:
```
Comment (carolromero) [inline docs/en/modules/admin/pages/templates/proposal_answers.adoc]:
```suggestion
All the templates appear in the proposal answers templates list:
```

File: docs/en/modules/admin/pages/templates/proposal_answers.adoc [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+* *Template*: name of the proposal answer template.
+* *Internal state*: state given to proposals when using this template. 
+* *Add constraint*: proposal component where you can use this template. 
+* *Created at*: date of creation.
+
+You can edit a template and its questions at any moment.
```
Comment (carolromero) [inline docs/en/modules/admin/pages/templates/proposal_answers.adoc]:
```suggestion
You can edit a template and its fields at any moment. 
```

**Review by carolromero — CHANGES_REQUESTED** [review]
Some changes

**Comment by paulinebessoles** [comment]
@carolromero This PR is ready to review and merge

**Pull request merged**